dev-manuals/test-plans/lecturer-interface/u-lecturer-interface-22-navigate-course-list.md

```diff
@@ -0,0 +1,34 @@
+# Lecturer-Interface: Navigate course list (`u.lecturer-interface-22`)
+
+
+Version: V1.0, 30.08.2022 \
+Author: Aaron Schmid
+
+## Description
+
+The course list can be navigated
+
+## Precondition
+
+At least three course are created.
+
+## Postcondition
+
+A different course is selected.
+
+## Typical procedure
+
+1. Click <kbd>down</kbd>
+2. The second course is selected
+3. Click <kbd>up</kbd>
+4. The first course is selected
+
+## Alternative procedures
```

Comment:
```suggestion
## Alternative procedures

2.1. no course below - ? \
4.1. no course above - ?
```
(**Please fill in the question marks before applying this suggestion**, i.e. with a new comment)